.github/workflows/test.yml

```diff
@@ -104,7 +104,7 @@ jobs:
     name: Test all
     runs-on: ubuntu-latest
     needs: build
-    if: ${{ github.event.inputs.regenerate-disks != 'true' }}
+    if: ${{ github.event.inputs.regenerate-disks != 'true' && github.event.inputs.run-full-sync != 'true' }}
```

Comment:
This should be

`${{ github.event.inputs.regenerate-disks != 'true' || github.event.inputs.run-full-sync != 'true' }}`

Reply:
If we use or `||`, we get these outcomes:
* regenerate-disks: false /  run-full-sync: false - tests run
* regenerate-disks: false /  run-full-sync: true - tests run
* regenerate-disks: true /  run-full-sync: false - tests run
* regenerate-disks: true /  run-full-sync: true - tests do not run

But we want this outcome using and `&&`:
* regenerate-disks: false /  run-full-sync: false - tests run
* regenerate-disks: false /  run-full-sync: true - tests do not run
* regenerate-disks: true /  run-full-sync: false - tests do not run
* regenerate-disks: true /  run-full-sync: true - tests do not run

Reply:
My bad, wasn't considering the negation from `!= 'true'` as in the `workflow_call` we compare `booleans` and here `true` is returned as a string instead of a `boolean` type.

That GHA issue got me all mixed up
- https://github.com/actions/runner/issues/1483